worker/activities/todo.go

```diff
@@ -0,0 +1,48 @@
+package activities
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+	"worker/config"
+	"worker/models"
+)
+
+func GetTodosDueTodayActivity(ctx context.Context) (models.TodosDueTodaySuccessResponse, error) {
+	fmt.Println(",,,,,,,,,,,,,,,activity,,,,,,,,,,,,,,,")
```

Comment:
can be removed or use proper log statement

Reply:
:(